probatus/feature_elimination/feature_elimination.py

```diff
@@ -556,7 +550,7 @@ def fit(
                     sample_weight=sample_weight,
                     **shap_kwargs,
                 )
-                for train_index, val_index in self.cv.split(current_X, self.y, groups)
+                for train_index, val_index in self.cv.split(current_X, self.y)
```

Comment:
We need groups for the implementation since this guarantees a cv split which keeps the groups in mind (and thus conceptually allows us to alleviate some bias introduced when we don’t consider for these groups).